tests/queries/test_q.py

```diff
@@ -88,6 +85,12 @@ def test_deconstruct_nested(self):
         self.assertEqual(args, (Q(price__gt=F('discounted_price')),))
         self.assertEqual(kwargs, {})
 
+    def test_deconstruct_not_subscriptable(self):
+        q = Q(False)
```

Comment:
This is not a supported usage. Please add a test with conditional expressions, e.g. `Exists()`.

Reply:
Done